Sources/PostgresNIO/New/Messages/Describe.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  File.swift
+//
+//
+//  Created by Fabian Fett on 08.01.21.
+//
+
+extension PSQLFrontendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLFrontendMessage {
```